webservices/tasks/refresh.py

```diff
@@ -20,9 +22,10 @@ def refresh():
             manage.refresh_itemized()
             manage.refresh_materialized()
             download.clear_bucket()
+            slack_message = '*Success* nightly updates for {0} completed'.format(env.get_credential(NEW_RELIC_APP_NAME))
+            utils.post_to_slack(slack_message, '#bots')
```

Comment:
Just confirming that `NEW_RELIC_APP_NAME` is being used to identify the app here.  Also, as noted above, should we set a default environment variable for the channel name to post to?

Reply:
I was a way to get text as to which environment it was running on without adding more variables